source/guides/guides/parallelization.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Parallelization
+---
+
+{% note info %}
+# {% fa fa-graduation-cap %} What You'll Learn
+
+- 
+{% endnote %}
+
+# Overview
+
+If your project has a large number of tests, it can take a large amount of time for tests to complete running when run on one machine serially. Spreading your tests across many virtual machines and running tests in parallel can save your team time and money when running tests in Continuos Integration.
+
+Cypress can run recorded tests in parallel across multiple virtual machines.
+
+This guide assumes you already have your project running and recording within Continuous Integration. If you haven't set up your project yet, check out our Continuous Integration guide.
```

Comment:
Consider hyperlinking to the CI guide: makes navigating these excellent docs even more of a pleasure.

```
check out our {% url 'Continuous Integration' continuous-integration %} guide.
```